WooCommerce/Classes/ViewRelated/Orders/OrdersSplitViewWrapperController.swift

```diff
@@ -74,6 +74,10 @@ private extension OrdersSplitViewWrapperController {
     }
 
     func showSecondaryView(_ viewController: UIViewController) {
+        // added to remove double details presented bug #11752 https://github.com/woocommerce/woocommerce-ios/pull/11753#discussion_r1463020153
+        // - white debugging noticed that ordersViewController.navigationController had multiple orders in the view controllers list
+        ordersViewController.navigationController?.popToRootViewController(animated: false)
```

Comment:
nice, I noticed the double details when tapping on a deep link while another order is already shown in the collapsed state because the navigation stack is merged when the split view is collapsed. I tried finding a way to customize how the merged navigation stack is separated from collapsed to expanded state, but haven't had success looking at the [split view delegate methods](https://developer.apple.com/documentation/uikit/uisplitviewcontrollerdelegate#3596702). Popping the order list navigation stack to the root is a good solution.